Unix/tools/auth_tests_setup.sh

```diff
@@ -1,8 +1,7 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
we might don't have bash on some Unix build system.

Reply:
required on unix systems (solaris, aix, hpux)